File: ProjectTwo/randomGraphs.R
# This is an example of using iGraph library in R to plot a random graph
#
# Author: Huo Chen
#
# Revison History
# ============================================================================
# First commit                                                       9/22/1988
g <- sample_gnp(20,0.5)  # Generate a binomial graph with 20 nodes
plot(g)

File: ProjectTwo/randomGraphs.m
%{
This is an example to create a binomial random graph in MATLAB

Author: Huo Chen

Revision History
=========================================================================gr
First Commit                                                   9/22/2016
%}

% Create a random edge matrix with 20 nodes
edge=random('Binomial',1,0.5,20,20);
edge(edge==0)=-1;
edge=abs(edge'+edge);
edge(logical(eye(20)))=0;
G=graph(edge~=0);

% Plot the graph

plot(G)